### PR TITLE
Initial description and example api

### PR DESCRIPTION
**Pull request by MartinSchmidt**
This pull request adds a description OriginStamp project,  and example API for issuing GCs.

[Link to the readme](https://github.com/project-origin/issuer/blob/description/README.md)

### CODE REVIEW

**Review by Mike4Elia — APPROVED** [review]
Looks good. 

One minor remarks in the 'API Example', There is an example to create the recipient with a Public Key, but there's no mention where this key is coming from.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,71 @@
...
+
+The name `OriginStamp` is a play on words,
+where `Origin` links to the project's name, while `Stamp` evokes the idea of officially marking or certifying something,
+emphasizing the service's role in issuing certificates.
+
+## tl:dr
+
+In the Project-Origin Electricity implementation, consists of a fungible and non-fungible digital assets, where the non-fungible assets are called Guarantees of Origin (GCs). The fungible assets are called GC slices.
+
+The GCs are issued by an issuing body and are stored in a registry. And an owner keeps track of their ownership of GC slices in a wallet. The process of issuing a GC is a complex process that involves multiple systems, to simplify this process, the OriginStamp is created.
```
Comment (sahma19) [inline README.md:13]: Have you thought about the implications, regarding how people will feel when issuance is handled by another entity that is not the TSO?

Right now energinet, being the TSO for Denmark, is also issuing body, using the certificates subsystem.

Having the TSO themselves act as issuing body creates a perception of validity, that cannot be replicated, should one choose to let another entity act as issuing body. While I do understand that the certificates are issued, using measurement data, directly from the TSO (energinet), I still believe that the perceived validity of those issued GC's will decrease. Would like to discuss this further, should you have the time.

**Review by sahma19 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tnickelsen) [inline README.md:13]: The GCs are not stored in the registry are they? Thought they were registered there and then stored in the wallet

File: doc/example-api.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+OK 200
+
+```json
+{
+    "status": "pending",
+    "id": "f5d8ee39-0f64-4a8f-9d4e-e25b8f3a5b3f",
+}
+```
+
+## Create a recipient
```
Comment (CodeReaper) [inline doc/example-api.md:48]: Can you update a recipient? I am thinking about if a person moves their wallet, or their wallet provider changes their endpoint, etc.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,71 @@
...
+
+OpenSource project to create a issuer service to simplify issuing certificates on the Project Origin platform.
+
+The name `OriginStamp` is a play on words,
+where `Origin` links to the project's name, while `Stamp` evokes the idea of officially marking or certifying something,
+emphasizing the service's role in issuing certificates.
+
+## tl:dr
+
+In the Project-Origin Electricity implementation, consists of a fungible and non-fungible digital assets, where the non-fungible assets are called Guarantees of Origin (GCs). The fungible assets are called GC slices.
```
Comment (CodeReaper) [inline README.md:11]: It is unclear what the word "consists" refers to

File: README.md (hunk 1 above)
Comment (MartinSchmidt) [reply README.md]: All the public data, as in, the transactions are stored and available on the registries.

The wallet stores the ownes private data regarding their GCs.

File: doc/example-api.md (hunk 2 above)
Comment (MartinSchmidt) [reply doc/example-api.md:48]: Sure, will probably come, this was more to show a list of example APIs to begin highlighting what could be possible, not a complete list.

File: README.md (hunk 3 above)
Comment (MartinSchmidt) [reply README.md]: rewrote it

File: README.md (hunk 1 above)
Comment (MartinSchmidt) [reply README.md]: The stamp service should be thought of as a service an TSO or other issuing body can self-host and use. 
It won't enable others to issue certificates, unless that is accepted within the given network (Energy Track and Trace).

So think of it as a tool to make the process easier for a Issuing body, not something other people would be able to use.

**Review by sahma19 — APPROVED** [review]

**Pull request merged**